_posts/2019-07-12-barcelona-announcement.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: post
+title: "Barcelona announcement"
+---
+Barcelona, a sunny city on the Mediterranean sea, capital of Catalonia, the land of *castellers* and *pà amb tomaca*, is considered one of the top tech hubs in Europe as it’s holding more than 1300 startups and generating almost 14000 jobs.
+
+![Barcelona](/assets/posts/barcelona-city.jpg)
+
+As a touristic destination, Barcelona is known for its beautiful architecture like Sagrada Familia, Casa Batlló, Ciutadella’s park and Palau de la Musica; for its delicious gastronomy and for its multicultural and open personality.
+
+There are lots of ways to enjoy the city: you can go to the beach, have a hike on some green areas like Park Güell or just take a walk through the different neighborhoods in order to experience how the influence of History has affected the city.
+
+## Save the date!
+
+We have finally a date for this year's edition!
+
+Rustfest 2019 will start on **November 9th** (Saturday) and will end on **November 12th** (Tuesday).
+
+As usual, this edition will have two first days full of talks and workshops and two additional days (*aka Impl Days*) where you will be able to get together with other rustaceans, meet in smaller groups and work on your own community projects.
+
+## Mixing things a little bit
+
+Although we are keeping the same length in terms of days we have decided to change things a little bit and try something different.
+
+This year, the conference format will change and instead of a first day of talks and a second day of workshops we're going to try to split the days into talks in the morning and workshops in the afternoon.
+
+We took into account the feedback we received from previous editions and we believe this can work well for most of you. Indeed, this will allow us to have more talks and workshops and reduce the intensity of a monothematic format day.
+
+## Venue
+
+Rustfest Barcelona 2019 is going to be hold in the [Vèrtex building at Campus Nord](https://www.upc.edu/campusnord/ca/espais/auditori) belonging to [UPC - Universitat Politècnica de Catalunya](https://www.upc.edu/en).
+
+![Vèrtex Auditorium](/assets/posts/barcelona-vx_auditori.jpg)
+
+We'll be using the two level **Vèrtex auditorium** with capacity for 477 people and access for people with reduced mobility at any of the two levels which we think will make a great venue for the amazing talks we'll be hosting this year. ;D
+
+The workshops and the *Impl Days** will take place in some of the [university's classrooms](https://www.upc.edu/campusnord/ca/espais/aules-vertex) which are fully equiped to host workshops of almost any kind.
+
+We will have also available a chilling area for resting between sessions.
```

Comment:
By "chilling area", are you talking about the quiet room or merely a chilling area? Those two should be distinguished in case.

Reply:
This I'm not sure as I don't know the venue. Hopefully on Monday I will. @ferranrego maybe can clarify this?

Reply:
Yes, that would be good to find out before you mention it :) 
The quiet room should have a different function than merely a chilling area.
Please read here: https://adacamp.org/adacamp-toolkit/quiet-room/
On another note, it might be good to mention free childcare service here, too!

Reply:
Sure! Let's confirm this with @ferranrego before saying anything.

Reply:
By chilling area I meant that it was like a little garden in the inside, where there's some places to sit.
At least the times I went to that venue, the organizers made it like that!

Reply:
I will mention childcare and quiet space, too. I think @ferranrego is not working today.

Reply:
@RafaelaWetzel if you are ok with this you can resolve it.

Reply:
I'm not that familiar with GitHub yet, what does it imply to "resolve it"?

Reply:
Just hides the conversation and makes it clear that it's not an issue anymore